packages/components/src/__actions__/Button/v3/_docs/Button.stories.tsx

```diff
@@ -19,80 +21,55 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Playground: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+  render: ({ children, ...otherProps }) => (
+    <Button {...otherProps}>{children}</Button>
   ),
 }
 
 export const ButtonWithIcon: Story = {
```

Comment:
To match pattern of next story?

```suggestion
export const ButtonWithIconStart: Story = {
```

Reply:
This ^